admin/c2cgeoportal_admin/__init__.py

```diff
@@ -1,13 +1,21 @@
+import c2cwsgiutils.pyramid
 from pyramid.config import Configurator
+from c2cwsgiutils.health_check import HealthCheck
 
 
 def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
     config = Configurator(settings=settings)
+    config.include(c2cwsgiutils.pyramid.includeme)
     config.include('pyramid_jinja2')
     config.include('c2cgeoform')
     config.include('c2cgeoportal_commons')
     config.include('.routes')
     config.scan()
+
+    health_check = HealthCheck(config)
+    health_check.add_url_check('http://localhost/')
```

Comment:
might not work with paster serve

Reply:
Correct, http://localhost:6543/health_check displays `Failed to establish a new connection: [Errno 111] Connection refused'`